app/src/main/java/org/sample/MainActivity.kt

```diff
@@ -0,0 +1,161 @@
+package com.example.androidsample
```

Comment:
We should review what we leave in the sample app, since there's stuff that we may not use